src/config.ts

```diff
@@ -154,3 +155,5 @@ export const deploys = {
 
 const stableDeploys = [...deploys.production, deploys.staging]
 export const isStableDeploy = stableDeploys.some(url => window.location.origin === url)
+
+export const getAppTitle = (t: TFunction) => process.env.REACT_APP_TITLE ?? t('appName')
```

Comment:
This should be only `REACT_APP_TITLE`, no default.

Reply:
And probably appName can be removed, if it is not being used anywhere else

Reply:
Yes, I have done both now.